model/converter/thrift/jaeger/sampling_from_domain.go

```diff
@@ -72,6 +72,9 @@ func convertPerOperationFromDomain(s *api_v2.PerOperationSamplingStrategies) *sa
 		for i, k := range s.PerOperationStrategies {
 			r.PerOperationStrategies[i] = convertOperationFromDomain(k)
 		}
+	} else {
+		// Initialize to an empty array so that using json.Marshal results in [] instead of null.
+		r.PerOperationStrategies = make([]*sampling.OperationSamplingStrategy, 0)
 	}
```

Comment:
I think you can simply remove the if statement and only keep its content
```
        perOp := s.GetPerOperationStrategies()
        // Want an empty array by default so that using json.Marshal results in [] instead of null.
        r.PerOperationStrategies = make([]*sampling.OperationSamplingStrategy, len(perOp))
        for i, k := range perOp {
            r.PerOperationStrategies[i] = convertOperationFromDomain(k)
        }
```

Reply:
D'oh! ty!